test/Prelude.purs

```diff
@@ -66,8 +68,15 @@ checkFixture filepath fixturePath = do
   fixturecontent <- FS.readTextFile fixturePath
   filecontent `Assert.shouldEqual` fixturecontent
 
-checkResultAndOutputs :: Maybe FilePath -> Maybe FilePath -> (Either ExecError ExecResult -> Boolean) -> Either ExecError ExecResult -> Aff _
-checkResultAndOutputs maybeOutFixture maybeErrFixture resultFn execResult = do
+plusDependencies :: Array String -> Config -> Config
+plusDependencies deps config = config
+  { package = config.package <#> \p -> p { dependencies = p.dependencies <> (Dependencies $ Map.fromFoldable $ map mkDep deps) }
+  }
+  where
+  mkDep p = Tuple (unsafeFromRight $ PackageName.parse p) Nothing
+
+checkResultAndOutputs' :: Maybe String -> Maybe FilePath -> (Either ExecError ExecResult -> Boolean) -> Either ExecError ExecResult -> Aff _
+checkResultAndOutputs' maybeOutStr maybeErrStr resultFn execResult = do
```

Comment:
Thoughts on renaming this to something else? The `'` isn't much of a difference and the type signature is the same as `checkResultAndOutputs`. So, I see this as an easy gotcha when writing tests in the future.

Reply:
Good point. How's `checkResultAndOutputsStr`?

Reply:
Done.